components/radio/group.tsx

```diff
@@ -84,16 +91,50 @@ export default class RadioGroup extends React.Component<RadioGroupProps, any> {
   }
   render() {
     const props = this.props;
-    const children = !props.children ? [] : React.Children.map(props.children, (radio: any) => {
-      if (radio && radio.type && (radio.type.__ANT_RADIO || radio.type.__ANT_RADIO_BUTTON) && radio.props) {
-        return React.cloneElement(radio, assign({}, radio.props, {
-          onChange: this.onRadioChange,
-          checked: this.state.value === radio.props.value,
-          disabled: radio.props.disabled || this.props.disabled,
-        }));
-      }
-      return radio;
-    });
+
+    let children: React.ReactChildren[] | React.ReactElement<any>[];
+
+    // 如果存在 options, 优先使用
+    if (props.options && props.options.length > 0) {
+      children = props.options.map((option, index) => {
+        if (typeof option === 'string') { // 此处类型自动推导为 string
+          return (
+            <Radio
+              key={index}
+              disabled={this.props.disabled}
+              value={option}
+              onChange={this.onRadioChange}
+              checked={this.state.value === option}
+            >
+              {option}
+            </Radio>
+          );
+        } else { // 此处类型自动推导为 { label: string value: string }
+          return (
+            <Radio
+              key={index}
+              disabled={option.disabled || this.props.disabled}
+              value={option.value}
+              onChange={this.onRadioChange}
+              checked={this.state.value === option.value}
+            >
+              {option.label}
+            </Radio>
+          );
+        }
+      });
+    } else {
+      children = !props.children ? [] : React.Children.map(props.children, (radio: any) => {
+        if (radio && radio.type && (radio.type.__ANT_RADIO || radio.type.__ANT_RADIO_BUTTON) && radio.props) {
+          return React.cloneElement(radio, assign({}, radio.props, {
```

Comment:
This will have conflict with @yesmeck 's refactoring https://github.com/ant-design/ant-design/commit/ab6e521ea221f19f7fd3604237c9380fb70c7cec